src/perf/lib/PerfServer.h

```diff
@@ -18,12 +18,12 @@
 
 class PerfServer : public PerfBase {
 public:
-    PerfServer(const QUIC_CREDENTIAL_CONFIG* CredConfig) :
+    PerfServer(const QUIC_CREDENTIAL_CONFIG* CredConfig, uint16_t Port) :
         Engine(TcpAcceptCallback, TcpConnectCallback, TcpReceiveCallback, TcpSendCompleteCallback),
         Server(&Engine, CredConfig, this) {
         CxPlatZeroMemory(&LocalAddr, sizeof(LocalAddr));
         QuicAddrSetFamily(&LocalAddr, QUIC_ADDRESS_FAMILY_UNSPEC);
-        QuicAddrSetPort(&LocalAddr, PERF_DEFAULT_PORT);
+        QuicAddrSetPort(&LocalAddr, Port);
```

Comment:
`-bind:0.0.0.0:port` should work as another way. But regardless, instead of doing it at this layer, this kind of code should go in `PerfServer::Init` to update `LocalAddr`.